packages/r/rambda.json

```
@@ -18,13 +18,16 @@
     "url": "https://github.com/selfrefactor"
   },
   "license": "MIT",
-  "npmName": "rambda",
-  "npmFileMap": [
-    {
-      "basePath": "",
-      "files": [
-        "webVersion.js"
-      ]
-    }
-  ]
+  "autoupdate": {
+    "source": "npm",
+    "target": "rambda",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*.js?(.map)"
```

```suggestion
          "*.js"
```
There are no map files provided in the package, so there is no point in adding that to the glob?